### PR TITLE
D kleverov

### PR DESCRIPTION
**Pull request by denklewer**
Выполнил задание 4, исправил баг в тесте 4

### CODE REVIEW

File: src/main/java/com/epam/university/java/core/task004/Task004Impl.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                }
+            }
+            if (firstCopy[i] != null) {
+                result[resultIndex++] = firstCopy[i];
+            }
+        }
+        for (int i = 0; i < secondCopy.length; i++) {
+                result[resultIndex++] = secondCopy[i];
+            }
+
```
Comment (aabarmin) [inline src/main/java/com/epam/university/java/core/task004/Task004Impl.java]: Do you really need empty lines here?)

File: src/test/java/com/epam/university/java/core/task004/Task004Test.java [hunk 2]
```diff
@@ -120,6 +120,7 @@ public void testUnionThird() throws Exception {
         };
         final String[] second = {};
         final String[] target = {
+                "",
```
Comment (aabarmin) [inline src/test/java/com/epam/university/java/core/task004/Task004Test.java]: Hmm. I think, here should be an empty array.

Comment (aabarmin) [inline src/test/java/com/epam/university/java/core/task004/Task004Test.java:122]: You are right, array of two empty strings. Fixed

**Comment by aabarmin** [comment]
You should update your branch from master to get travis CI and style checker.

**Comment by denklewer** [comment]
I have added it, but it fails checking for uncompleted tasks

**Pull request merged**